lib/core/src/lib/form/services/widget-visibility.service.ts

```diff
@@ -134,7 +134,7 @@ export class WidgetVisibilityService {
         } else if (visibilityObj.rightType === WidgetTypeEnum.field) {
             valueFound = this.getFormValue(form, visibilityObj.rightValue);
         } else {
-            if (moment(visibilityObj.rightValue, 'YYYY-MM-DD', true).isValid()) {
+            if (isMatch(visibilityObj.rightValue, 'yyyy-MM-dd')) {
```

Comment:
this value comes from the backend server and should remain the same not to break existing forms

Reply:
It is a condition check. No values are being changed. What is it that  you suggest I should do with it?

Reply:
the right value needs to be converted from moment probably